src/fdcache_entity.h

```diff
@@ -154,6 +155,7 @@ class FdEntity : public std::enable_shared_from_this<FdEntity>
             const std::lock_guard<std::mutex> lock(fdent_lock);
             return path;
         }
+        std::string GetPathWithoutLock() const { return path; }
```

Comment:
Since we don't want to expose the public method without locking, should remove this method.